src/injector/src/main.rs

```diff
@@ -130,7 +130,7 @@ fn start_seed_registry() {
                     // on Content-Length, we respond the same as a GET
                     accept!(
                         request,
-                        "application/vnd.oci.image.manifest.v1+json" => {
+                        "application/vnd.docker.distribution.manifest.v2+json" => {
```

Comment:
My rust is rusty, but would it make sense to capture/perform on both `...oci.image...` and `...docker.distribution...`?

Reply:
The change is occurring because we moved to using crane vs a manually hacked source OCI for the seed registry. This passed with containerd, but seems to be sensitive with docker as CRI. I don't think supporting both would be necessary as this is purely an init package change: the rust injector logic + the oci objects in the same package.

Reply:
I was that because when we're [creating the init package](https://github.com/defenseunicorns/zarf/blob/a0b13947b9b345b09ce23e6d848bd587012f1bc3/src/pkg/packager/create.go#L93) we are string replacing the `docker.distribution` with `oci.image`: https://github.com/defenseunicorns/zarf/blob/00736dd9d00d7b24f838612dc8c260dc5c41ee4c/src/internal/packager/images/pull.go#L186.


I think it makes sense to have endpoints that way we can handle both 'request types' but I'm not sure I 100% understand what went wrong here.

Reply:
It's a result of the change to crane vs hand-rolled OCI image. This is a permanent fix.